src/common/transformations/src/transformations/op_conversions/convert_reduce_to_reshape.cpp

```diff
@@ -0,0 +1,67 @@
+
+#include "itt.hpp"
+
+ngraph::pass::ConvertReduceMeanToReshape::ConvertReduceMeanToReshape() {
+    MATCHER_SCOPE(ConvertReduceMeanToReshape);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<opset1::ReduceMean>(
+            {pattern::any_input(pattern::has_static_shape()), pattern::wrap_type<opset1::Constant>()},
+            pattern::has_static_shape()),
+        matcher_name);
+    register_matcher(m, convert_reduce_to_reshape<opset1::ReduceMean>());
+}
+
+ngraph::pass::ConvertReduceSumToReshape::ConvertReduceSumToReshape() {
+    MATCHER_SCOPE(ConvertReduceSumToReshape);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<opset1::ReduceSum>(
+            {pattern::any_input(pattern::has_static_shape()), pattern::wrap_type<opset1::Constant>()},
+            pattern::has_static_shape()),
+        matcher_name);
+    register_matcher(m, convert_reduce_to_reshape<opset1::ReduceSum>());
+}
+
+ngraph::pass::ConvertReduceMaxToReshape::ConvertReduceMaxToReshape() {
+    MATCHER_SCOPE(ConvertReduceMaxToReshape);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<opset1::ReduceMax>(
+            {pattern::any_input(pattern::has_static_shape()), pattern::wrap_type<opset1::Constant>()},
+            pattern::has_static_shape()),
+        matcher_name);
+    register_matcher(m, convert_reduce_to_reshape<opset1::ReduceMax>());
+}
+
+ngraph::pass::ConvertReduceMinToReshape::ConvertReduceMinToReshape() {
+    MATCHER_SCOPE(ConvertReduceMinToReshape);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<opset1::ReduceMin>(
+            {pattern::any_input(pattern::has_static_shape()), pattern::wrap_type<opset1::Constant>()},
+            pattern::has_static_shape()),
+        matcher_name);
+    register_matcher(m, convert_reduce_to_reshape<opset1::ReduceMin>());
+}
+
+ngraph::pass::ConvertReduceLogicalAndToReshape::ConvertReduceLogicalAndToReshape() {
+    MATCHER_SCOPE(ConvertReduceLogicalAndToReshape);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<opset1::ReduceLogicalAnd>(
+            {pattern::any_input(pattern::has_static_shape()), pattern::wrap_type<opset1::Constant>()},
+            pattern::has_static_shape()),
+        matcher_name);
+    register_matcher(m, convert_reduce_to_reshape<opset1::ReduceLogicalAnd>());
+}
+
+ngraph::pass::ConvertReduceLogicalOrToReshape::ConvertReduceLogicalOrToReshape() {
+    MATCHER_SCOPE(ConvertReduceLogicalOrToReshape);
+    auto m = std::make_shared<ngraph::pattern::Matcher>(
+        ngraph::pattern::wrap_type<opset1::ReduceLogicalOr>(
+            {pattern::any_input(pattern::has_static_shape()), pattern::wrap_type<opset1::Constant>()},
+            pattern::has_static_shape()),
+        matcher_name);
+    register_matcher(m, convert_reduce_to_reshape<opset1::ReduceLogicalOr>());
+}
```

Comment:
nit: please add an empty line at the end

Reply:
Added